src/v/config/configuration.cc

```diff
@@ -1962,7 +1962,15 @@ configuration::configuration()
       "Interval, in seconds, of how often a message informing the operator "
       "that unsafe strings are permitted",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
-      300s) {}
+      300s)
+  , kafka_memory_share_for_fetch(
+      *this,
+      "kafka_memory_share_for_fetch",
+      "The share of kafka subsystem memory that can be used for fetch read "
+      "buffers, as a fraction of kafka subsystem memory amount",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::user},
+      0.3,
```

Comment:
Thinking about what the default should be... there's no "right answer" here, but maybe 0.5 makes sense, on the basis of optimizing for 1:1 produce:consume workloads?

I'm anticipating being asked "why" at some point, and needing some kind of basis to explain the %ge we choose.

Reply:
Good point, was thinking of that too... essentially this value is "1 - (how much memory of kafka s/sys is saved from being used by the fetch op)". So saving 70% by default might sound a bit too agressive... I think 0.5 is a better choice still.